### ISSUE TITLE
flux-core docker image for 0.26.0 pulls the incorrect version

### ISSUE THREAD

**Issue by FrankD412** [flux-framework/flux-core#3786]
Not 100% sure if this is the right place to post this, so apologies in advanced if it is.

When testing Maestro using Docker, I ran into the following when testing with flux-core@0.26.0. I have a Dockerfile that distinctly asks for `fluxrm/flux-core:centos7-v0.26.0` but the build seems to have a version of flux-core@0.25.0 installed. I discovered this error when Maestro threw an exception that it couldn't find the specified version of Flux, and expected it to work as it had worked in the past. I tried CentOS8, and got the same result, so I'm wondering if the hashed 0.25.0 version was intentional.


See below the output and Dockerfile:

```
(3.9.2) frank@Friday  ~/code/Python/maestro/maestrowf   bugfix/flux_walltime ±  docker build -f ./docker/flux/0.26.0/Dockerfile -t flux_test .
[+] Building 73.9s (15/15) FINISHED
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                                                                          0.1s
 => => transferring dockerfile: 38B                                                                                                                                                                                                                                                                           0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                                                                                             0.1s
 => => transferring context: 2B                                                                                                                                                                                                                                                                               0.0s
 => [internal] load metadata for docker.io/fluxrm/flux-core:centos7-v0.26.0                                                                                                                                                                                                                                  16.4s
 => [ 1/10] FROM docker.io/fluxrm/flux-core:centos7-v0.26.0@sha256:1ada355a9e7ab640301ab4f8de0cf1dfbcf3bda870d477254e05f9563ec52709                                                                                                                                                                           0.0s
......
 => exporting to image                                                                                                                                                                                                                                                                                        1.1s
 => => exporting layers                                                                                                                                                                                                                                                                                       1.0s
 => => writing image sha256:3c5bdff384d93b1d67d43d1ebfebe4cea57dc7b1eafad3e01a28748e6ac6ea08                                                                                                                                                                                                                  0.0s
 => => naming to docker.io/library/flux_test                                                                                                                                                                                                                                                                  0.0s
(3.9.2) frank@Friday  ~/code/Python/maestro/maestrowf   bugfix/flux_walltime ±  docker run -it flux_test
ƒ(s=1,d=0) [fluxuser@76a34c5031d5 ~]$ flux version
commands:               0.25.0-90-gfc2bf21
libflux-core:           0.25.0-90-gfc2bf21
libflux-security:       0.4.0
broker:                 0.25.0-90-gfc2bf21
FLUX_URI:               local:///tmp/flux-RUb9LF/local-0
build-options:          +caliper+hwloc==1.11.0
ƒ(s=1,d=0) [fluxuser@76a34c5031d5 ~]$ exit
exit
```

The Dockerfile is checked in [here](), and reproduced below:
```
FROM fluxrm/flux-core:centos7-v0.26.0
ENV PATH="/home/fluxuser/.local/bin:$PATH"

RUN curl -sL https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.5.tar.gz | tar xz && \
    cd ./openmpi-4.0.5 && \
    ./configure && \
    make && \
    sudo make install

COPY . /home/fluxuser/maestrowf
WORKDIR /home/fluxuser/maestrowf
RUN echo `which python3` && python3 --version
RUN echo `which pip3` && pip3 --version
RUN pip3 install -U --user pip
RUN echo "$PWD" && ls -la && pip3 install --user .
RUN pip3 install -U ipython
WORKDIR /home/fluxuser

LABEL maintainer="Francesco Di Natale dinatale3@llnl.gov"
```

**Comment by SteVwonder** [comment]
Thanks @FrankD412 . I think this is a duplicate of https://github.com/flux-framework/flux-core/issues/3689

tl;dr: it is actually v0.26.0, the git tags just weren't properly pulled down to hardcode that version correctly in the binary.  It is fixed with docker images v0.27.0+

**Comment by FrankD412** [comment]
Ah, it does appear that way -- thanks @SteVwonder
